examples/fluent-bit/cloudwatchlogs/task-definition-cloudwatch_logs.json

```diff
@@ -0,0 +1,42 @@
+{
+	"family": "firelens-example-cloudwatch",
```

Comment:
consider revising to firelens-example-cloudwatchlogs

Reply:
thanks/fixed